token.go

```diff
@@ -63,11 +79,12 @@ Add a configuration file with your CircleCI token, like this:
     token = "aabbccddeeff00"
 
 Go to https://circleci.com/account/api if you need to create a token.
-`, filename, rcFilename)
+`, strings.Join(checkedLocations, " or "))
 		return "", err
 	}
+	defer f.Close()
 	var c CircleConfig
-	_, err = toml.DecodeReader(f, &c)
+	_, err = toml.DecodeReader(bufio.NewReader(f), &c)
```

Comment:
The bufio.NewReader is a good practice for anything that reads or writes from a file. toml.DecodeReader might do a bunch of small reads which isn't good for performance. bufio.NewReader will do one big read and then copy from there as necessary.